distribution/server-dist/assembly.xml

```diff
@@ -33,13 +33,28 @@
             <includes>
                 <include>**/module.xml</include>
             </includes>
+            <!-- unused resteasy providers eat up memory -->
+            <excludes>
```

Comment:
Removing modules will probably cause issues with patching in RH-SSO. It will also not have any effect on RPM distro and OpenShift images as they are created as overlays on-top of EAP rather than full distros.

Reply:
does changing an EAP module.xml file have similar patching issues?

Reply:
Nevermind, jaxrs subsystem automatically includes those modules if they exist.  I thought they might be included via a module dependency.

BTW, would be really curious to know if anybody actually uses our RPM or OpenShift images.